ReactAndroid/src/main/jni/first-party/fb/log.cpp

```diff
@@ -28,8 +28,9 @@ int fb_printLog(int prio, const char *tag,  const char *fmt, ...) {
   va_end(va_args);
   if (gLogHandler != NULL) {
       gLogHandler(prio, tag, logBuffer);
+  } else {
+    __android_log_write(prio, tag, logBuffer);
```

Comment:
We don't need to write the log to logcat in case we have handler as this is a handler responsibility now.
If we do we will have duplicate logs.